### PR TITLE
feat(fluid-text-input): adds skeleton variant

### LINKED ISSUE #12080: Fluid inputs: TextInput - add skeleton state
Add a skeleton state for fluid text input, originally left out of spec.

image

### PR DESCRIPTION
**Pull request by aledavila**
Closes #12080

Adds Fluid Text Input Skeleton 

#### Changelog

**New**

- adds new skeleton 

#### Testing / Reviewing

Make sure Fluid Text input is working as expected. Make sure skeleton matches spec

### CODE REVIEW

File: packages/react/src/components/FluidTextInput/index.js [hunk 1]
```diff
@@ -7,3 +7,5 @@
 
 export default from './FluidTextInput';
 export FluidTextInput from './FluidTextInput';
+
+export { default as FluidTextInputSkeleton } from './FluidTextInput.Skeleton';
```
Comment (tay1orjones) [inline packages/react/src/components/FluidTextInput/index.js:11]: I think this needs to be exposed in the main index.js as well and the public api snapshot regenerated.

**Review by tay1orjones — CHANGES_REQUESTED** [review]
Looks good to me, just one small change.

Also, should the skeleton animate? I don't see it animating the ribbon/gradient shimmer effect in the storybook

File: packages/react/src/components/FluidTextInput/index.js (hunk 1 above)
Comment (aledavila) [reply packages/react/src/components/FluidTextInput/index.js:11]: Yea good catch I updated the animations as well @tay1orjones

**Review by aagonzales — APPROVED** [review]
It looks good to me!

**Review by jnm2377 — APPROVED** [review]

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**